includes/promoted-jobs/class-wp-job-manager-promoted-jobs-status-handler.php

```diff
@@ -102,4 +126,24 @@ private function get_site_feed_url() {
 		return add_query_arg( 'site', home_url( '', 'https' ), WP_Job_Manager_Helper_API::get_wpjmcom_url() . '/wp-json/promoted-jobs/v1/site/jobs' );
 	}
 
+	/**
+	 * Updates the interval option with the value from the response header from the feed.
+	 *
+	 * @param array|\WP_Error $response The HTTP response from the feed.
+	 * @param string          $header_name The name of the header to check for the interval value.
+	 * @param string          $option_name The name of the option to update with the interval value.
+	 * @return void
+	 */
+	private function update_interval( $response, $header_name, $option_name ) {
+		$headers = wp_remote_retrieve_headers( $response );
+		if ( isset( $headers[ $header_name ] ) ) {
+			$header = $headers [ $header_name ];
+			if ( 'false' === $header ) {
+				delete_option( $option_name );
+			} elseif ( rest_is_integer( $header ) ) {
+				update_option( $option_name, (int) $header, false );
+			}
+		}
+	}
+
```

Comment:
This is the logic to update the option according to the response headers. If the value is the magic `false`, we delete the option, otherwise we verify if it is an integer and, if it is, we update the value on the database